addons/web/static/src/views/fields/dynamic_widget/dynamic_model_field_selector_char.js

```diff
@@ -83,10 +84,11 @@ export const dynamicModelFieldSelectorChar = {
             type: "string",
         },
     ],
-    extractProps({ options }, dynamicInfo) {
+    extractProps({ required, options }, dynamicInfo) {
```

Comment:
required is already available and evalued in dynamicInfo.
There is the same problem with the readonly props

Reply:
Hey @kebeclibre
Any other request or can we send this fix? 😄